pipecat-client-android-small-webrtc-transport/src/main/java/ai/pipecat/client/small_webrtc_transport/SmallWebRTCTransport.kt

```diff
@@ -114,7 +170,7 @@ class SmallWebRTCTransport internal constructor(
                                     disconnect()
                                 }
 
-                                "renegotiate" -> negotiate()
+                                "renegotiate" -> negotiate(transportParams)
```

Comment:
is there any reason this should use `connectParams` instead? i.e. is it possible for the connectParams to change between now and when a renegotiation occurs and therefore should use the updated params?